statistics.md

```diff
@@ -476,22 +333,127 @@ After disabling the `ANALYZE` configuration persistence feature, TiDB does not c
+
+It is recommended to ensure that all tables/indexes (and partitions) utilize statistics collection from the same version. Version 2 is recommended, however, it is not recommended to switch from one version to another without a justifiable reason such as an issue experienced with the version in use. A switch between versions might take a period of time when no statistics are available until all tables have been analyzed with the new version, which might negatively affect the optimizer plan choices if statistics are not available.
+
+Examples of justifications to switch might include - with Version 1, there could be inaccuracies in equal/IN predicate estimation due to hash collisions when collecting Count-Min sketch statistics. Solutions are listed in the [Count-Min Sketch](#count-min-sketch) section. Alternatively, setting `tidb_analyze_version = 2` and rerunning `ANALYZE` on all objects is also a solution. In the early release of Version 2, there was a risk of memory overflow after `ANALYZE`. This issue is resolved, but initially, one solution was to `set tidb_analyze_version = 1` and rerun `ANALYZE` on all objects.
 
-- Minimum value: should be greater than the maximum memory usage when TiDB collects statistics from the table with the most columns. An approximate reference: when TiDB collects statistics from a table with 20 columns using the default configuration, the maximum memory usage is about 800 MiB; when TiDB collects statistics from a table with 160 columns using the default configuration, the maximum memory usage is about 5 GiB.
-- Maximum value: should be less than the available memory when TiDB is not collecting statistics.
+To prepare `ANALYZE` for switching between versions:
+
+- If the `ANALYZE` statement is executed manually, manually analyze every table to be analyzed.
+
+    ```sql
+    SELECT DISTINCT(CONCAT('ANALYZE TABLE ', table_schema, '.', table_name, ';'))
+    FROM information_schema.tables JOIN mysql.stats_histograms
+    ON table_id = tidb_table_id
+    WHERE stats_ver = 2;
+    ```
+
+- If TiDB automatically executes the `ANALYZE` statement because the auto-analysis has been enabled, execute the following statement that generates the [`DROP STATS`](/sql-statements/sql-statement-drop-stats.md) statement:
+
+    ```sql
+    SELECT DISTINCT(CONCAT('DROP STATS ', table_schema, '.', table_name, ';'))
+    FROM information_schema.tables ON mysql.stats_histograms
+    ON table_id = tidb_table_id
+    WHERE stats_ver = 2;
+    ```
+
+- If the result of the preceding statement is too long to copy and paste, you can export the result to a temporary text file and then perform execution from the file like this:
+
+    ```sql
+    SELECT DISTINCT ... INTO OUTFILE '/tmp/sql.txt';
+    mysql -h ${TiDB_IP} -u user -P ${TIDB_PORT} ... < '/tmp/sql.txt'
+    ```
+
+## View statistics
+
+You can view the `ANALYZE` status and statistics information using the following statements.
 
 ### View `ANALYZE` state
```

Comment:
```suggestion
### `ANALYZE` state
```

Reply:
reason: to use the same style as other subsection titles in **View statistics**
![image](https://github.com/pingcap/docs/assets/79440533/674746e7-60e4-41f5-99c6-50607aa00ff9)